docs/source/en/main_classes/logging.md

```diff
@@ -71,6 +71,19 @@ verbose to the most verbose), those levels (with their corresponding int values
 
 By default, `tqdm` progress bars will be displayed during model download. [`logging.disable_progress_bar`] and [`logging.enable_progress_bar`] can be used to suppress or unsuppress this behavior.
 
+## `logging` vs `warnings`
```

Comment:
Can you add an example of what this means for us now? 
(calling logger with deprecation warning for example?) for all devs !

Reply:
You don't call logger with deprecation warnings, you just use `warnings.warn`. But now, you can capture those and eliminate them from the logging system simply.